src/lib.rs

```diff
@@ -88,55 +88,29 @@
 //! * `diesel-uuid` enables integration with Diesel's UUID support, this is
 //!   only tested on postgres, PRs welcome for other DBs.
 
-extern crate base64;
-#[cfg(feature = "diesel")]
-#[macro_use]
-extern crate diesel_derive_newtype;
-#[macro_use]
-extern crate error_chain;
-extern crate inlinable_string;
-#[macro_use]
-extern crate lazy_static;
-extern crate uuid;
-
-#[cfg(all(test, feature = "diesel-uuid"))]
-#[macro_use]
-extern crate diesel;
-#[cfg(all(test, feature = "diesel-uuid"))]
-#[cfg(all(test, feature = "serde"))]
-#[macro_use]
-extern crate serde_derive;
-#[cfg(all(test, feature = "serde"))]
-#[macro_use]
-extern crate serde_json;
-
 use std::convert::From;
-use std::str::FromStr;
 use std::fmt::{Debug, Display, Formatter, Result as FmtResult};
+use std::str::FromStr;
 
-use uuid::Uuid;
-use base64::{CharacterSet, Config, LineWrap};
 use base64::display::Base64Display;
+use base64::engine::general_purpose::URL_SAFE_NO_PAD;
+use base64::Engine;
 use inlinable_string::inline_string::InlineString;
+use uuid::Uuid;
 
-use errors::{ErrorKind, ResultExt};
+#[cfg(feature = "diesel-uuid")]
+#[macro_use]
+extern crate diesel_derive_newtype;
+
+use crate::errors::{ErrorKind, ResultExt};
 
 mod errors;
 #[cfg(feature = "serde")]
 mod serde_impl;
 
-lazy_static! {
-    static ref B64_CONFIG: Config = Config::new(
-        CharacterSet::UrlSafe,
-        false, // pad?
-        false, // trim whitespace?
```

Comment:
I guess the base64 crate no longer handles whitespace itself (https://github.com/marshallpierce/rust-base64/issues/163) which seems legitimate.